actions/loadconfig.yaml

```diff
@@ -28,17 +28,25 @@ parameters:
     config_file:
         type: "string"
         description: "Absolute path to configuration file on local system that will be sent to the device"
```

Comment:
We should mention that
- we need to specify either config_file or config_text parameter
- config_file takes precedence if both given

```
description: "Absolute path to configuration file on local system that will be sent to the device. Need to specify either config_file or config_text. If both given, config_file takes precedence."